DuckDuckGo/TabViewController.swift

```diff
@@ -419,11 +424,20 @@ class TabViewController: UIViewController {
     }
     
     public func load(url: URL) {
+        webView.stopLoading()
+        if jsAlertController.isShown {
+            jsAlertController.dismiss(animated: false)
+        }
+
```

Comment:
This flag validation is being done in multiple places, what do you think about creating a func for it? Something like this:

```swift
  private func dismissJSAlertIfNecessary(animated: Bool) {
      if jsAlertController.isShown {
          jsAlertController.dismiss(animated: animated)
      }
  }